Brainfuck/R/bf.R

```diff
@@ -0,0 +1,59 @@
+library(stringr)

```

Comment:
Which functions used are part of this library?

Reply:
Removed